style_guide.md

```diff
@@ -1,26 +1,100 @@
+*Adapt and adopt [OCDS style guidance for property descriptions](https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#field-and-code-descriptions) ?*
 
-- Object and property titles: first word to be upper case. E.g. 'Interest' or 'Ownership-or-control statement'.
+'The top level of the schema is split between properties and definitions. The latter contains objects that may be re-used, by reference, in multiple locations across the schema. Each of these can be thought of as a “Class”, and its name is capitalized accordingly. Whenever you consider that an object or structure might be re-used in a different area of the standard, it should be included in definitions.'
 
-- In descriptions, properties should be referred to by their property name in backquotes. E.g. "\`id\` is required". Property values, whether example free text strings or values from a codelist, should appear in single quotes. E.g. "the given name for Johann Sebastian Bach is 'Johann Sebastian'".
+I think we are already doing this in practice for at least some 'definitions' via the 'components' file - e.g 'Addresses'? Would the adoption of this guidance just to be to state that explicitly? Or would this be some other change to the schema itself?
 
-- In descriptions or other longer text fields, use plain text only, without markup (eg. no \[Markdown\](https://en.wikipedia.org/wiki/Markdown) or similar syntax). Where URLs are necessary, they can be placed in brackets by the appropriate word or phrase or at the end of the sentence. 
+*Markup within descriptions?* 
 
-- Codelists codes to be lowercase when single words, or camelCase when multiple words.
+*How should a codelist be referred to? I suggest by its inferred title, capitalised. e.g. "...choose an item from the Unspecified Reason codelist...". And it would be translated for internationalisation.* - in the OCDS schema codelists are referred to in descriptions using camel case and aren't translated there. The titles in the codelist page are translated though. 
 
-**WIP notes**
-
-*Adapt and adopt [OCDS style guidance for property descriptions](https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#field-and-code-descriptions) ?*
+*How should a code be referred to? By its code or by its title? If by code, this wouldn't be translated. Perhaps use single quotes (e.g. "...use 'placeOfBirth' to refer to an address...") Then translation guidance could instruct translators to retain the code in the text but follow it with the translated code title (e.g. "... utilise 'placeOfBirth' (lieu de naissance) pour faire référence à une adresse...". Similar guidance would work for dealing with property names in translation (see above).* 
 
-*Markup within descriptions?*
+*Should we use 'object' or 'block' or even 'component' to refer to JSON objects?* - object?
 
-## BODS documentation
+## Markdown style guide 
```

Comment:
Markdown refers to the way that this Handbook page is formatted. RestructuredText is a different thing. Prob need to keep this section as 'Documentation styleguide' or similar.

Reply:
Maybe 'Documentation formatting'?